spec/std/crystal/static_list_spec.cr

```diff
@@ -0,0 +1,299 @@
+require "spec"
+require "crystal/static_list.cr"
+
+private struct TestedObject
+  property value : Int32
+  property link : Crystal::StaticList
+
+  def initialize(@value : Int32)
+    @link = uninitialized Crystal::StaticList
+  end
+end
+
+private class TestedObject2
+  property value : Int32
+  property link : Crystal::StaticList
+
+  def initialize(@value : Int32)
+    @link = uninitialized Crystal::StaticList
+  end
+end
+
+describe "Crysta::StaticList" do
```

Comment:
`Crystal`.

Reply:
This typo can be avoided by removing the quotes.